src/commands/UpdatedCommand.js

```diff
@@ -16,7 +16,7 @@ export default class UpdatedCommand extends Command {
 
   execute(callback) {
     const formattedUpdates = this.updates
-      .map((update) => `- ${update.package.name}`)
+      .map((update) => `- ${update.package.name}${update.package.isPrivate()?" (private)":""}`)
```

Comment:
Looks like that linter rule got dropped accidentally.  [Re-added](https://github.com/lerna/lerna/commit/309029397fc6f4a7bac7020af2ba4313c4bf1e4b) and fixed this up.

Nice catch @hzoo!